di_website/templates/publications/publication_foreword_page.html

```diff
@@ -0,0 +1,80 @@
+{% extends "layouts/default.html" %}
+
+{% load string_utils wagtailcore_tags responsive wagtailimages_tags %}
+
+{% block hero %}
+{% include 'includes/heroes/hero-pubs-short.html' %}
+{% endblock hero %}
+
+{% block colour %}body--{{page.colour}}{% endblock %}
+
+{% block body %}
+
+{% include 'publications/chapter-nav/nav.html' with page=page.get_parent.specific menu_label='Contents' chapter_number=0 %}
+
+<section class="section section--large">
+
+{% for block in page.content %}
+
+  <div class="chapter-block-wrapper">
+    <div class="row row--narrow">
+      <div class="larger-center">
+        <div class="chapter-block__item">
+            {% if block.block_type == 'case_study' %}
+            <div class="pullOutBlock">
+              {% endif %}
+
+              <div class="row row--narrow">
+                {% if forloop.first %}
+                {% if page.authors|length > 0 %}
+                  <div class="chapter-block_content">
+                    {% include 'includes/partials/multi-authors.html' with authors=page.authors %}
+                  </div>
+                {% endif %}
```

Comment:
Does the foreword have authors? I don't think so ... clean this up and only keep what's required... and try to add proper indenting, even with blocks & loops ... through the entire template.

Reply:
Updated